<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="framerate" value="15" />
    <param name="pixel_format" value="yuyv" />
    <param name="io_method" value="mmap"/>
    <param name="autofocus" value="false"/>
    <param name="focus" value="7"/>
    <param name="brightness" value="125"/>
    <param name="camera_name" value="chessboard"/>
    <param name="camera_frame_id" value="chessboard"/>

    <remap from="/usb_cam/image_raw" to="/camera/color/image_raw" />

  </node>
  <!--node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node-->
</launch>